tests/test_cookiecutter_xblock.py

```diff
@@ -109,12 +109,12 @@ def test_quality(options_baked):
     """Run quality tests on the given generated output."""
     sh.make('upgrade')
     sh.pip('install', '-r', 'requirements/base.txt')
```

Comment:
Can this be moved to the fixture as in the other tests?

Reply:
Yes, done.